setup.py

```diff
@@ -513,6 +513,7 @@ def setup_package():
         project_urls={
             "Bug Tracker": "https://github.com/numpy/numpy/issues",
             "Documentation": get_docs_url(),
+            "Release Notes" = "https://numpy.org/doc/stable/release",
```

Comment:
Can you please drop this change? This won't be used and will give a merge conflict in another PR.